### PR TITLE
#160536360 Implement functionality for admin to view menus

### PR DESCRIPTION
**Pull request by abdulfataiaka**
#### What does this PR do?
- Adds functionality for admin to view all menus on the application

#### Description of Task to be completed?
- Remove in memory data used
- Make request to `mockapi` for menus
- Update test for new implementations

#### How should this be manually tested?
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ft-admin-view-menus-160536360`
- Run these commands `npm run bundle` and `npm run dev` to setup app
- Log in as an `Admin`, navigate to `admin dashboard`, select `Menus Icon` on the `side nav`.
- Ensure to be redirected to a page where you see listing of all menus
- Run the command `npm run test-dev` to ensure all tests are passing

#### What are the relevant pivotal tracker stories?
- [#160536360](https://www.pivotaltracker.com/story/show/160536360)

#### Screenshots (If Applicable)
screen shot 2018-10-10 at 5 11 03 pm

### CODE REVIEW

File: client/src/helpers/configAxios.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import axios from 'axios';
 
 const configAxios = (token) => {
-  axios.defaults.headers['x-access-token'] = token || "";
```
Comment (Charpell) [inline client/src/helpers/configAxios.js]: Have you verified that these dosen't break any of the endpoint after removal?

Comment (abdulfataiaka) [reply client/src/helpers/configAxios.js]: Yes, as the only place it exists now is in this file.

Comment (Charpell) [reply client/src/helpers/configAxios.js]: Okay

File: client/src/components/Admin/Menus/Index.jsx [hunk 2]
```diff
@@ -122,53 +232,58 @@ class Menus extends Component {
...
+   * @description render side and protein listing
+   *
+   * @param { Array } itemsAvailable
+   * @param { Integer } optionsCanPick
+   *
+   * @memberof Menus
+   * 
+   * @returns { JSX }
+   */
+  renderListPurge = (itemsAvailable, optionsCanPick) => (
```
Comment (monsieurkayode) [inline client/src/components/Admin/Menus/Index.jsx]: Can you explain what this method does?

File: client/src/actions/admin/menuItemsAction.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      const menus = response.data;
+
+      dispatch(fetchMenusSuccess(menus));
+      dispatch(fetchMenusLoading(false));
+    })
+    .catch(() => {
+      // Can be updated to dispatch error with
+      // custom error messages instead of null
+
+      dispatch(fetchMenusError(null));
```
[the remark this answers is not in the document]
Comment (abdulfataiaka) [reply client/src/actions/admin/menuItemsAction.js]: Yes.

**Comment by abdulfataiaka** [comment]
@Charpell and @monsieurkayode, please review as all comments has been addressed.

**Pull request merged**